src/hooks/useConnection.tsx

```diff
@@ -17,6 +17,7 @@ type ConnectionState = {
   network: Network | null;
   address: Address | null;
   error: Error | null;
+  isConnected: boolean;
```

Comment:
It would be awesome if in the future we can import this `useConnection` hook into other dApps that we build, seeing as this logic would be pretty generically used across web3 dApps